PFCalEE/userlib/src/HGCSSGeometryConversion.cc

```diff
@@ -119,24 +128,82 @@ void HGCSSGeometryConversion::initialiseSquareMap(TH2Poly *map, const double xym
   
 }
 
-void HGCSSGeometryConversion::initialiseHoneyComb(const double xymin, const double side){
-  initialiseHoneyComb(hexagonMap(),xymin,side,true);
+void HGCSSGeometryConversion::initialiseHoneyComb(const double width, const double side){
+  initialiseHoneyComb(hexagonMap(),width,side,true);
   fillXY(hexagonMap(),hexaGeom);
 }
 
-void HGCSSGeometryConversion::initialiseHoneyComb(TH2Poly *map, const double xymin, const double side, bool print){
+void HGCSSGeometryConversion::initialiseHoneyComb(TH2Poly *map, const double width, const double side, bool print){
   //xstart,ystart,side length,
-  double d=sqrt(3.)*side;
-  unsigned nx=static_cast<unsigned>(xymin*2./d);
-  unsigned ny=static_cast<unsigned>(xymin*4./(3.*side));
+
+  // Center a cell at (x,y)=(0,0) and ensure coverage up to/past width/2 in all 4 directions,
+  // assuming each cell is lying on a side.
+
+  unsigned ncellwide=11;
```

Comment:
I think ncellwide should be given as argument (instead of "width" which is not used...) and to avoid hardcoding the "11" ...